buildlib/az-distro-release.yml

```diff
@@ -1,6 +1,10 @@
+parameters:
+  arch:
+  demands: []
+
 jobs:
-  - job: distro_release
-    displayName: distro
+  - job: distro_release_${{ parameters.arch }}
+    displayName: Distro ${{ parameters.arch }}
```

Comment:
Currently, the job title appears as `Distro aarch64 ubuntu20_cuda11_aarch64`
Can we rename it to just "ubuntu20_cuda11_aarch64" or similar?
![image](https://user-images.githubusercontent.com/2255631/232413582-2ab098f3-bed8-4e84-9f92-8ce57ec4fe89.png)

Reply:
![image](https://user-images.githubusercontent.com/57558538/232429206-2d97aa2d-c202-47ed-b728-188bc077f422.png)
Close enough?

Reply:
Much better indeed

![image](https://user-images.githubusercontent.com/57558538/232440604-9ac43acf-3de5-40e4-a398-6342dab32c24.png)